docs/DeveloperGuide.adoc

```diff
@@ -768,6 +792,21 @@ e.g. `loanbook.storage.StorageManagerTest`
 ..  Hybrids of unit and integration tests. These test are checking multiple code units as well as how the are connected together. +
 e.g. `loanbook.logic.LogicManagerTest`
 
+[WARNING]
+====
+The tests within the `add` command parsers do not
```

Comment:
Do not what?

Reply:
Good catch. That's weird though, I think I failed to copy and paste one of the lines from my draft. Thanks for pointing this out!